tests/test_downloader.py

```diff
@@ -651,3 +651,28 @@ def parse(
             "<locals>.TestSpider.parse() missing 1 required positional argument: 'page2'"
         )
     assert expected_msg in caplog.text
+
+
+@inlineCallbacks
+def test_parse_callback_NO_CALLBACK() -> None:
+    """See: https://github.com/scrapinghub/scrapy-poet/issues/118"""
+
+    try:
+        from scrapy.http.request import NO_CALLBACK
+    except ImportError:
+        pytest.skip("NO_CALLBACK not available in Scrapy < 2.8")
+
+    with MockServer(EchoResource) as server:
+
+        class TestSpider(BaseSpider):
+            start_urls = [server.root_url]
+
+            def start_requests(self):
+                yield Request(server.root_url, callback=NO_CALLBACK)
```

Comment:
Is it possible to support regular spiders now, e.g. with start_urls without start_requests?

Reply:
Addressed in 19b7cf4

Reply:
Addressed in 6c5895e, but some of the asyncio toxenvs are still failing.

Reply:
I've created a separate PR in #129 to address the unrelated CI failures.